src/app/core/entity-components/entity-utils/dynamic-form-components/edit-text-with-autocomplete/edit-text-with-autocomplete.component.html

```diff
@@ -20,11 +20,24 @@
       ></app-display-entity>
     </mat-option>
   </mat-autocomplete>
-
+  <mat-hint>
```

Comment:
```suggestion
  <mat-hint *ngIf="!autocompleteDisabled">
```

Reply:
When editing an existing record, the autocomplete is disabled and therefore this should not be shown